### ISSUE TITLE
Pre-work Checklist: Developer: Gayle Martin

### ISSUE THREAD

**Issue by w1ld-r** [hackforla/website#6246]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by w1ld-r** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took about an hour to finish the prework up to ETA/availability, and about two hours to finish my first issue and submit my first PR. I plan to attend my first meetings this Thursday, Sunday, and Tuesday.

**Comment by HackforLABot** [comment]
## Activity Log: gaylem
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- gaylem issue closed by PR 6300: https://github.com/hackforla/website/issues/3861#event-8871295890 at 03/28/2023, 07:28 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6246 at 02/05/2024, 07:30 PM PST
- gaylem assigned: https://github.com/hackforla/website/issues/6246 at 02/05/2024, 07:30 PM PST
- gaylem assigned: https://github.com/hackforla/website/issues/6050 at 02/06/2024, 10:01 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/6050#issuecomment-1930488166 at 02/06/2024, 10:04 AM PST
- gaylem opened: https://github.com/hackforla/website/pull/6260 at 02/06/2024, 11:16 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/6246#issuecomment-1930603459 at 02/06/2024, 11:20 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6260#issuecomment-1930931161 at 02/06/2024, 03:17 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6260#issuecomment-1932367945 at 02/07/2024, 08:03 AM PST
- gaylem PR was merged: https://github.com/hackforla/website/pull/6260#event-11754415939 at 02/08/2024, 10:43 PM PST
- gaylem issue closed by PR 6260: https://github.com/hackforla/website/issues/6050#event-11754416032 at 02/08/2024, 10:44 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6273#issuecomment-1937025101 at 02/10/2024, 06:53 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6273#pullrequestreview-1873725212 at 02/10/2024, 06:59 AM PST
- gaylem assigned: https://github.com/hackforla/website/issues/3861#issuecomment-1913460430 at 02/10/2024, 07:04 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/3861#issuecomment-1937037326 at 02/10/2024, 07:07 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6273#pullrequestreview-1874381939 at 02/11/2024, 06:44 AM PST
- gaylem opened: https://github.com/hackforla/website/pull/6281 at 02/12/2024, 08:29 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/3861#issuecomment-1939085012 at 02/12/2024, 08:29 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6281#issuecomment-1939085398 at 02/12/2024, 08:29 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6279#issuecomment-1939104924 at 02/12/2024, 08:35 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6279#pullrequestreview-1875736041 at 02/12/2024, 08:56 AM PST
- gaylem assigned: https://github.com/hackforla/website/issues/5985 at 02/13/2024, 02:16 PM PST
- gaylem unassigned: https://github.com/hackforla/website/issues/5985 at 02/13/2024, 02:19 PM PST
- gaylem closed as completed: https://github.com/hackforla/website/issues/6246#event-11796224619 at 02/13/2024, 06:59 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6292#issuecomment-1943028187 at 02/13/2024, 07:26 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6292#pullrequestreview-1880528276 at 02/14/2024, 06:59 AM PST
- gaylem opened: https://github.com/hackforla/website/pull/6300 at 02/15/2024, 06:09 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6281#issuecomment-1946174697 at 02/15/2024, 06:11 AM PST
- gaylem closed: https://github.com/hackforla/website/pull/6281#event-11816041206 at 02/15/2024, 06:11 AM PST
- gaylem PR was merged: https://github.com/hackforla/website/pull/6300#event-11824529073 at 02/15/2024, 09:27 PM PST
- gaylem issue closed by PR 6300: https://github.com/hackforla/website/issues/3861#event-11824529257 at 02/15/2024, 09:27 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6292#pullrequestreview-1886489594 at 02/16/2024, 11:15 PM PST
- gaylem assigned: https://github.com/hackforla/website/issues/5991 at 02/19/2024, 11:47 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/5991#issuecomment-1953077225 at 02/19/2024, 11:48 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/5991#issuecomment-1954693240 at 02/20/2024, 09:24 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6324#issuecomment-1954709017 at 02/20/2024, 09:33 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6324#pullrequestreview-1891088131 at 02/20/2024, 09:44 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6324#issuecomment-1954738156 at 02/20/2024, 09:47 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6322#issuecomment-1954745951 at 02/20/2024, 09:50 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6322#pullrequestreview-1891139190 at 02/20/2024, 10:10 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6324#issuecomment-1955979696 at 02/20/2024, 10:29 PM PST
- gaylem issue closed as completed: https://github.com/hackforla/website/issues/5991#event-11872298828 at 02/20/2024, 11:58 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6352#issuecomment-1957991614 at 02/21/2024, 01:30 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6351#issuecomment-1957992013 at 02/21/2024, 01:30 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6352#pullrequestreview-1894479186 at 02/21/2024, 01:46 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6351#pullrequestreview-1894492788 at 02/21/2024, 01:56 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6383#issuecomment-1969137305 at 02/28/2024, 06:43 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6362#issuecomment-1969137399 at 02/28/2024, 06:43 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6383#pullrequestreview-1907521136 at 02/28/2024, 02:06 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6362#pullrequestreview-1907532707 at 02/28/2024, 02:15 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6390#issuecomment-1970022639 at 02/28/2024, 02:24 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6390#pullrequestreview-1907611416 at 02/28/2024, 02:58 PM PST
- gaylem assigned: https://github.com/hackforla/website/issues/4856 at 02/28/2024, 03:10 PM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1970072440 at 02/28/2024, 03:11 PM PST
- gaylem opened: https://github.com/hackforla/website/issues/6397 at 02/29/2024, 08:30 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1973463593 at 03/01/2024, 08:10 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/6397#issuecomment-1973682123 at 03/01/2024, 10:13 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1974826346 at 03/02/2024, 07:14 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1974827105 at 03/02/2024, 07:16 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6436#issuecomment-1982012691 at 03/06/2024, 03:12 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6435#issuecomment-1982012789 at 03/06/2024, 03:12 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6436#pullrequestreview-1921026304 at 03/06/2024, 03:23 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6435#pullrequestreview-1921033414 at 03/06/2024, 03:30 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6433#issuecomment-1982034322 at 03/06/2024, 03:32 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6433#pullrequestreview-1921046079 at 03/06/2024, 03:44 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6431#issuecomment-1982058298 at 03/06/2024, 03:48 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6417#issuecomment-1982058447 at 03/06/2024, 03:48 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6431#pullrequestreview-1921067108 at 03/06/2024, 04:09 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6417#pullrequestreview-1921103998 at 03/06/2024, 04:41 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6432#pullrequestreview-1921114991 at 03/06/2024, 04:51 PM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6432#pullrequestreview-1921192959 at 03/06/2024, 05:42 PM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6432#issuecomment-1982198269 at 03/06/2024, 05:56 PM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1983607252 at 03/07/2024, 06:21 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1983692505 at 03/07/2024, 07:00 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1983710459 at 03/07/2024, 07:07 AM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1983762477 at 03/07/2024, 07:29 AM PST
- gaylem submitted review: https://github.com/hackforla/website/pull/6432#pullrequestreview-1923880459 at 03/07/2024, 05:41 PM PST
- gaylem commented on: https://github.com/hackforla/website/issues/4856#issuecomment-1986419244 at 03/08/2024, 01:05 PM PST
- gaylem issue closed as completed: https://github.com/hackforla/website/issues/4856#event-12063543832 at 03/09/2024, 08:59 AM PST
- gaylem commented on: https://github.com/hackforla/website/pull/6458#issuecomment-1995309008 at 03/13/2024, 11:32 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6448#issuecomment-1995309327 at 03/13/2024, 11:32 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6448#pullrequestreview-1935074067 at 03/13/2024, 12:45 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6458#pullrequestreview-1935194353 at 03/13/2024, 01:57 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6448#pullrequestreview-1935258914 at 03/13/2024, 02:35 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/5059 at 03/19/2024, 07:17 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/5059#issuecomment-2008570476 at 03/19/2024, 07:19 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/5059#issuecomment-2010781311 at 03/20/2024, 03:38 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/5059#issuecomment-2012624495 at 03/21/2024, 08:24 AM PDT
- gaylem opened: https://github.com/hackforla/website/pull/6503 at 03/21/2024, 07:22 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6503#issuecomment-2017007368 at 03/24/2024, 05:18 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6503#issuecomment-2017018193 at 03/24/2024, 05:31 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6517#issuecomment-2021810056 at 03/26/2024, 07:34 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6516#issuecomment-2023058767 at 03/27/2024, 08:27 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6517#pullrequestreview-1964546393 at 03/27/2024, 12:17 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6516#issuecomment-2023827590 at 03/27/2024, 12:41 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6536#issuecomment-2023996097 at 03/27/2024, 02:10 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6536#pullrequestreview-1964837461 at 03/27/2024, 02:54 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6503#issuecomment-2024067640 at 03/27/2024, 03:02 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6503#issuecomment-2025861379 at 03/28/2024, 11:31 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6543#pullrequestreview-1967220328 at 03/28/2024, 01:30 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6536#issuecomment-2026082873 at 03/28/2024, 01:43 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6536#issuecomment-2026191584 at 03/28/2024, 02:49 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6548 at 03/29/2024, 07:49 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2027349578 at 03/29/2024, 07:55 AM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6378 at 03/29/2024, 01:18 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6548 at 03/29/2024, 01:18 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6536#pullrequestreview-1969570551 at 03/29/2024, 01:22 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6023#issuecomment-2027762141 at 03/29/2024, 02:41 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6378#issuecomment-2027777765 at 03/29/2024, 03:04 PM PDT
- gaylem opened: https://github.com/hackforla/website/pull/6551 at 03/30/2024, 11:51 AM PDT
- gaylem closed: https://github.com/hackforla/website/pull/6551#event-12301448408 at 03/30/2024, 11:52 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6551#issuecomment-2028441052 at 03/30/2024, 11:52 AM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6552 at 03/30/2024, 04:03 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6397 at 04/01/2024, 07:01 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6397#issuecomment-2030949807 at 04/01/2024, 07:11 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6552#issuecomment-2032248243 at 04/02/2024, 07:46 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2032252820 at 04/02/2024, 07:47 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6561#issuecomment-2039912865 at 04/05/2024, 07:12 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6561#pullrequestreview-1983340492 at 04/05/2024, 07:21 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6489#issuecomment-2039959726 at 04/05/2024, 07:34 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6489#issuecomment-2039996985 at 04/05/2024, 07:50 AM PDT
- gaylem closed as completed: https://github.com/hackforla/website/issues/6489#event-12375444222 at 04/05/2024, 07:50 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/5059#issuecomment-2040239751 at 04/05/2024, 09:42 AM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6397 at 04/05/2024, 12:52 PM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6378 at 04/05/2024, 12:54 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6561#issuecomment-2040562049 at 04/05/2024, 01:15 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6561#pullrequestreview-1984114500 at 04/05/2024, 01:15 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2040790973 at 04/05/2024, 04:52 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6552#issuecomment-2041522366 at 04/07/2024, 09:28 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6552#issuecomment-2048102569 at 04/10/2024, 10:27 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6506#event-12452039614 at 04/12/2024, 10:00 AM PDT
- gaylem closed by PR 6506: https://github.com/hackforla/website/issues/5840#event-12452039842 at 04/12/2024, 10:00 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6549#event-12452091697 at 04/12/2024, 10:05 AM PDT
- gaylem closed by PR 6549: https://github.com/hackforla/website/issues/6196#event-12452091940 at 04/12/2024, 10:05 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6625#issuecomment-2052202743 at 04/12/2024, 10:44 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6617#issuecomment-2052204419 at 04/12/2024, 10:45 AM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6058 at 04/12/2024, 10:50 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6058#issuecomment-2052212789 at 04/12/2024, 10:51 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6629#event-12454960625 at 04/12/2024, 04:08 PM PDT
- gaylem closed by PR 6629: https://github.com/hackforla/website/issues/6174#event-12454960725 at 04/12/2024, 04:08 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/4752 at 04/12/2024, 04:14 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6633 at 04/12/2024, 04:48 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/4752#issuecomment-2052715567 at 04/12/2024, 05:09 PM PDT
- gaylem closed as completed: https://github.com/hackforla/website/issues/4752#event-12455274972 at 04/12/2024, 05:13 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6637 at 04/12/2024, 05:42 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6397#issuecomment-2052785002 at 04/12/2024, 06:00 PM PDT
- gaylem closed as completed: https://github.com/hackforla/website/issues/6397#event-12455398453 at 04/12/2024, 06:00 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6397 at 04/12/2024, 06:03 PM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6548 at 04/14/2024, 05:07 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6649 at 04/15/2024, 09:12 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6442#issuecomment-2057349454 at 04/15/2024, 09:45 AM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6058 at 04/15/2024, 05:39 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6520#issuecomment-2058055037 at 04/15/2024, 06:02 PM PDT
- gaylem issue closed by PR 6503: https://github.com/hackforla/website/issues/5059#event-12515933176 at 04/17/2024, 04:38 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6649#issuecomment-2066714065 at 04/19/2024, 07:33 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6659#event-12545612980 at 04/19/2024, 03:47 PM PDT
- gaylem closed by PR 6659: https://github.com/hackforla/website/issues/6539#event-12545613065 at 04/19/2024, 03:47 PM PDT
- gaylem opened: https://github.com/hackforla/website/issues/6691 at 04/20/2024, 07:33 AM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6691 at 04/20/2024, 07:36 AM PDT
- gaylem opened: https://github.com/hackforla/website/pull/6692 at 04/20/2024, 11:21 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6692#issuecomment-2067768469 at 04/20/2024, 01:00 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6691#issuecomment-2067770732 at 04/20/2024, 01:10 PM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6520#issuecomment-2068167566 at 04/21/2024, 12:03 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6691#issuecomment-2072308517 at 04/23/2024, 06:20 AM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6649 at 04/23/2024, 10:58 AM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6649 at 04/24/2024, 12:08 PM PDT
- gaylem closed by PR 6520: https://github.com/hackforla/website/issues/6193#event-12629137915 at 04/26/2024, 04:05 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6649 at 04/28/2024, 10:14 AM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6649#issuecomment-2083137865 at 04/29/2024, 09:11 AM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6649 at 04/29/2024, 09:12 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6692#issuecomment-2085425110 at 04/30/2024, 07:03 AM PDT
- gaylem unassigned: https://github.com/hackforla/website/issues/6691 at 04/30/2024, 10:31 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6777#pullrequestreview-2032148319 at 04/30/2024, 11:03 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6777#issuecomment-2086337672 at 04/30/2024, 11:06 AM PDT
- gaylem commented on: https://github.com/hackforla/website/pull/6692#issuecomment-2087714803 at 04/30/2024, 04:31 PM PDT
- gaylem assigned: https://github.com/hackforla/website/issues/6691 at 04/30/2024, 04:34 PM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6777#pullrequestreview-2035573913 at 05/02/2024, 05:11 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6777#event-12682356185 at 05/02/2024, 05:12 AM PDT
- gaylem closed by PR 6777: https://github.com/hackforla/website/issues/6775#event-12682356399 at 05/02/2024, 05:12 AM PDT
- gaylem submitted review: https://github.com/hackforla/website/pull/6804#pullrequestreview-2037217017 at 05/02/2024, 06:26 PM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6808#event-12701068275 at 05/03/2024, 01:55 PM PDT
- gaylem closed by PR 6808: https://github.com/hackforla/website/issues/6705#event-12701068407 at 05/03/2024, 01:55 PM PDT
- gaylem closed by PR 6692: https://github.com/hackforla/website/issues/6691#event-12715027731 at 05/06/2024, 06:25 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6821#event-12770555020 at 05/10/2024, 07:31 AM PDT
- gaylem closed by PR 6821: https://github.com/hackforla/website/issues/6759#event-12770555264 at 05/10/2024, 07:31 AM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6841#event-12774263978 at 05/10/2024, 02:39 PM PDT
- gaylem closed by PR 6841: https://github.com/hackforla/website/issues/6717#event-12774264060 at 05/10/2024, 02:39 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2108793894 at 05/13/2024, 02:01 PM PDT
- gaylem commented on: https://github.com/hackforla/website/issues/6266#issuecomment-2111302403 at 05/14/2024, 04:03 PM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6913#event-13005179211 at 05/31/2024, 12:49 PM PDT
- gaylem closed by PR 6913: https://github.com/hackforla/website/issues/6735#event-13005179468 at 05/31/2024, 12:49 PM PDT
- gaylem merged: https://github.com/hackforla/website/pull/6900#event-13005187079 at 05/31/2024, 12:50 PM PDT
- gaylem closed by PR 6900: https://github.com/hackforla/website/issues/6731#event-13005187238 at 05/31/2024, 12:50 PM PDT
